docs/user-guide/amor/amor-reduction.ipynb

```diff
@@ -26,6 +26,7 @@
    "source": [
     "import scipp as sc\n",
     "from ess import amor\n",
+    "from ess.amor import data as amordata\n",
```

Comment:
Note that you can also
```suggestion
    "from ess.amor import data\n",
```
and then `amor.data` will be defined, i.e., you do not need to change anything below.

Reply:
Yeah that's better